resume/resume-2.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title>Nick Long Resume</title>
+	</head>
+	<body>
+		<style>
+	body {
+		background-color:rgb(243, 240, 234);
+		font-family: monospace;
+	}
+	#contact {
```

Comment:
use `#` for id attribute, `.` for class attributes

Reply:
I know, I just personally prefer class attr because I feel the notation is more streamlined.